README.md

```diff
@@ -48,6 +48,19 @@ Rototiller provides a Rake DSL addition called 'rototiller_task' which is a full
       end
     end
 
+    desc "do not include flag if value is nil or empty and not required"
```

Comment:
Yeah, I was trying to say that it is the final value after the default and override_env are evaluated.  What would be a more self explanatory word for that?  final value?